shopBuddy/shopBuddy/VCs/ItemDetailViewController.m

```diff
@@ -80,6 +89,7 @@ - (void) _callPricesAPI: (GlobalManager *)manager{
             [strongSelf.item syncPrices:prices];
             dispatch_async(dispatch_get_main_queue(), ^{
                 [strongSelf->activityIndicator stopAnimating];
+                strongSelf->doneLoading = YES;
```

Comment:
Risk using strongSelf inside inner async block, read comments above.
Issues with this might never manifest while you are debugging however potential app crashes are likely and unpredictable at runtime.